package.json

```diff
@@ -24,6 +24,7 @@
     "@material-ui/icons": "^3.0.2",
     "acorn": "^7.0.0",
     "axios": "^0.19.0",
+    "handlebars": ">=4.3.0",
```

Comment:
Will this pull major/minor version updates and maybe introduce problems?

Reply:
At the moment, our lock file has us covered. It's kind of surprising that Github's scanner doesn't just look at that (it kind of did for COTA UI, so who knows)

Reply:
I should say, within reason the lock file has us covered. We could just set to a minor within what the lock file has. I'll glance at that.